libs/books/feature/src/lib/reading-list/reading-list.component.html

```diff
@@ -8,17 +8,31 @@
       <div class="reading-list-item--details--author">
         {{ b.authors.join(',') }}
       </div>
+      <div *ngIf="b.finished" class="reading-list-item--details--author" data-testing="toggle-finish-date">
+        Finish reading at {{ b.finishedDate }}
+      </div>
     </div>
     <div>
       <button
         mat-icon-button
         color="warn"
         [attr.aria-label]="'Remove ' + b.title + ' from reading list'"
-        (click)="removeFromReadingList(b)"
+        (click)="removeFromReadingList(b)" data-testing="toggle-remove-book"
       >
         <mat-icon>remove_circle</mat-icon>
       </button>
     </div>
+    <div>
+      <button
+        mat-icon-button
+        data-testing="toggle-finish-book-cta"
```

Comment:
Change it to `data-testing="finish-book-cta"`

Reply:
Done